.vscode/launch.json

```diff
@@ -10,12 +10,26 @@
             "args": ["--extensionDevelopmentPath=${workspaceRoot}/vscode-lean4" ],
             "stopOnEntry": false,
             "sourceMaps": true,
-            "outFiles": [ "${workspaceRoot}/vscode-lean4/out/**/*.js" ],
+            "outFiles": [ "${workspaceRoot}/vscode-lean4/dist/**/*.js" ],
             "preLaunchTask": "watch",
             "debugWebviews": true,
             "rendererDebugOptions": {
                 "webRoot": "${workspaceRoot}/vscode-lean4/media"
             },
+        },
+        {
+            "name": "Debug Tests",
+            "type": "extensionHost",
+            "request": "launch",
+            "runtimeExecutable": "${execPath}",
+            "args": [
+                "--extensionDevelopmentPath=${workspaceFolder}/vscode-lean4",
+                "--extensionTestsPath=${workspaceFolder}/vscode-lean4/out/test/suite/toolchains",
```

Comment:
It looks like the proper way to setup `launch.json` is to have a separate configuration for every test suite, as in [here](https://github.com/microsoft/vscode-test/blob/ba93fae81c4be110755fda83d0d1e75a078c2b86/sample/.vscode/launch.json). Let's match ours with that.

Reply:
Fixed:
![image](https://user-images.githubusercontent.com/18707114/156104326-da8e72c9-a459-45ad-a91e-2085cbd86738.png)